test/integration/test_kubernetes_staging.py

```diff
@@ -101,6 +132,38 @@ def test_job_environment(self):
         assert job_env.some_env == '42'
 
 
+@integration_util.skip_unless_kubernetes()
+@integration_util.skip_unless_fixed_port()
+class KubernetesDependencyResolutionIntegrationTestCase(BaseJobEnvironmentIntegrationTestCase, MulledJobTestCases):
+
+    def setUp(self):
+        super(KubernetesDependencyResolutionIntegrationTestCase, self).setUp()
+        self.history_id = self.dataset_populator.new_history()
+
+    @classmethod
+    def setUpClass(cls):
+        # realpath for docker deployed in a VM on Mac, also done in driver_util.
+        cls.jobs_directory = os.path.realpath(tempfile.mkdtemp())
+        super(KubernetesDependencyResolutionIntegrationTestCase, cls).setUpClass()
+
+    @classmethod
+    def handle_galaxy_config_kwds(cls, config):
+        config["jobs_directory"] = cls.jobs_directory
+        config["file_path"] = cls.jobs_directory
+        config["job_config_file"] = job_config(DEPENDENCY_RESOLUTION_TEMPLATE, cls.jobs_directory)
+
+        config["default_job_shell"] = '/bin/sh'
+        # Disable tool dependency resolution.
+        config["tool_dependency_dir"] = "none"
+        config["enable_beta_mulled_containers"] = "true"
```

Comment:
That isn't necessary anymore, right ?

Reply:
Will fix it in a new PR 😆 - thanks @mvdbeek